core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -468,7 +471,7 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
      * `emergencyShutdown` as the governor who would call the function would also receive the fees.
      */
     function emergencyShutdown() external override onlyPreExpiration() onlyOpenState() {
-        require(msg.sender == _getFinancialContractsAdminAddress());
+        require(msg.sender == _getFinancialContractsAdminAddress(), "Only callable by admin(DVM vote)");
```

Comment:
why the parantheses "(DVM vote)"?

Reply:
I am trying to say the function is only callable by `admin`, which is initiated by a DVM vote resolution in favour of emergency shutdown. It's hard with 32 char limit haha. Any other suggestions I'm open to.